### DISCUSSION TITLE
Feedback branch missing in student repositories

### DISCUSSION THREAD

**Discussion by jheinzel** [community/community#113440] [GitHub Education]
### Hi everyone,

since January, I've noticed an issue with GitHub Classroom assignments. For a small fraction of student repositories, around 10 percent or less, no feedback branch or feedback pull request appears to be created. My assignments are individual assignments with a cutoff date, and feedback pull requests are activated. 

Because these problem only pertains to a small fraction of students, I assume concurrency issues might be the reason for this defect.

**Comment by samus-aran** [comment]
Hi @jheinzel  👋 

This seems to be a similar topic to the following discussion(s):

- https://github.com/orgs/community/discussions/106545

Please feel free to share your experiences and comments there. 👍  

We're closing this discussion because it appears to be a duplicate. Please re-open if you felt this was closed prematurely. Thank you for helping us maintain a productive and tidy community for all our members.
